CONTRIBUTING.md

```diff
@@ -120,20 +120,20 @@ $ docker-compose down
 
 ## Releasing
 
-Releasing a new version of this Gem:
-
-1. Update the version
-1. Run the release script
-
-### Update The Version
-
-- The version file (`lib/conjur-api/version.rb`) has been updated with an appropriate Semantic version number.
-- The `CHANGELOG.md` file has been updated to reflect the release version and appropriate release notes.
-
-Next, save -- but do not commit -- the changes above.
-
-### Run The Release Script
-
-```sh
-./bin/release
-```
+### Update the version and changelog
+
+1. Create a new branch for the version bump.
+1. Based on the unreleased content, determine the new version number and update
+    the [version.rb](lib/conjur-api/version.rb) file.
+1. Commit these changes - `Bump version to x.y.z` is an acceptable commit message - and open a PR
+   for review. Your PR should include updates to `lib/conjur-api/version.rb`, and
+    `CHANGELOG.md`.
+
+### Add a git tag
+
+1. Once your changes have been reviewed and merged into master, tag the version
+   using `git tag -a "vx.y.z" -m "vx.y.z release"`. Note this requires you to be able to sign releases.
+   Consult the [github documentation on signing commits](https://help.github.com/articles/signing-commits-with-gpg/)
+   on how to set this up. `vx.y.z release` is an acceptable tag message.
+1. Push the tag: `git push vx.y.z` (or `git push origin vx.y.z` if you are working
```

Comment:
pushing the tag will automatically publish to rubygems.org, right? might be good to call that out here